### DISCUSSION TITLE
Indicator for teleburned Ordinals

### DISCUSSION THREAD

**Discussion by jokie88** [ordinals/ord#1887] [Ideas]
Currently there is no way to easily distinguish between an inscribed copy of an NFT and a teleburned NFT. 

Would be cool if you could see that an inscription is teleburned in on the ordinals.com server.

**Comment by ghost** [comment]
Teleburn is a marketing word invented by some tik tok dancer, it means when somebody jumps ship from their current chain and transfers to the bitcoin blockchain.  This term did not catch on and has depreciated.  You can use reverse image search from google or TinEye dot com reverse image search service to try and check for previous references to a particular image, but there is no database or way to authenticate a "teleburn" (not a word) so it would be on an honor system and prone to scams.  You must verify the original owner of the NFT initiated a willing burn and return to the mother blockchain.

**Reply by slumericanBx** [reply]
@Thanks for the clarity. Almost teleburned my life savings away. o.0
